### ISSUE TITLE
Switch user during remote session

### ISSUE THREAD

**Issue by jsinoy** [Ylianst/MeshCentral#4409]
Hey everyone ,

We have implemented meshcentral and its working perfectly . I just wanna ask if why i cannot switch user during remote session ?

**Comment by Ylianst** [comment]
Are you talking about a remote desktop session? If you remote into a computer with many RDP sessions going on at the same time, you should see a screen allowing you to select what RDP session you wish to remote into. If this is what you are talking about, just disconnect and reconnect, you will get that list again and can pick a different session.

The screen will look like this (ignore the red circle).

![image](https://user-images.githubusercontent.com/1319013/185339371-de5f4877-38fc-45ca-9c6a-f8a39a24d628.png)

**Comment by jsinoy** [comment]
Hi Ylianst,

I'm using the normal connect session , while im connected to the computer . I can't seem to switch user on their desktop . It just freeze when i click the switch user button

**Comment by jsinoy** [comment]
During testing i found out that the client had successfully switch to login page , However on my end it just freeze

**Comment by jsinoy** [comment]
After further more testing,  i've found out that when "user consent" is disabled it is working fine , However when I enable "user consent"  , it just freezes . Cant view to the remote computer login page since it will ask for a user consent .

**Comment by jsinoy** [comment]
I think the problem since it requires "user consent"  i cant remote till the user accept the session . In similar case when i switch to another account during the remote session . The client successfully goes to the login page , but on the admin side the page stuck .  Disable "user consent" is not an option for now . Is it possible to have like an option to have a password like anydesk while "user consent" is enabled ?  In some cases if they cant accpt the "session" since they are still on the login page waiting for us to connect ?

**Comment by si458** [comment]
i have noticed this a few times,
when nobody is logged into the desktop and i then connect to it via MC and then login with say my user,
the screen window freezes and no screen updates,
BUT if you then click disconnect and then reconnect, the desktop reappears
i think it might be a bug?

**Comment by PathfinderNetworks** [comment]
Yes, this is a very common issue. Not just with MeshCentral.  Windows does something to the sessions when you switch users that causes this.  I think Ryan knows what causes it.  But, as si458 said, all you have to do is click Disconnect then Reconnect and you'll be back in.   Other remote tools get around this by detecting that the display is not updating and will automatically disconnect/reconnect (or something along those lines). 
Maybe something like that could be added to Mesh?
